egov-localization/README.md

```diff
@@ -0,0 +1,31 @@
+
+# <Localisation>
+
+This service provides localisation capacity to the Digit suite of services.
+
+### DB UML Diagram
+
+
+
+
+### Service Dependencies
+
+
+
+### Swagger API Contract
+
+
```

Comment:
API contract?

Reply:
API contract not available. we have to write new